### PR TITLE
chore: lint errors, and angular2 update

### PR DESCRIPTION
**Pull request by vanessayuenn**
- Fix lint errors
 - upgrade angular2 version to 2.0.0-alpha.45
 - remove angular2/core from webpack config since angualr2/angular2 already requires it.

Addresses #60

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -53,7 +53,6 @@ module.exports = {
       'reflect-metadata',
       // To ensure these modules are grouped together in one file
       'angular2/angular2',
-      'angular2/core',
       'angular2/router',
```
Comment (igor-ka) [inline webpack.config.js]: @vanessayuenn so router and http need to be included as well?

File: backend/adapters/angular2.ts [hunk 2]
```diff
@@ -296,22 +296,16 @@ export class Angular2Adapter extends BaseAdapter {
     switch (name) {
       case 'NgFor':
         return this._normalizeNgFor(state);
-        break;
       case 'NgIf':
         return this._normalizeNgIf(state);
-        break;
-      case "NgClass":
+      case 'NgClass':
```
Comment (igor-ka) [inline backend/adapters/angular2.ts]: removing the break and the spacing is a linting issue?

Comment (vanessayuenn) [inline backend/adapters/angular2.ts]: Yea the "break;" was unreachable code; and double quotes had to be changed to single quotes.

**Comment by igor-ka** [comment]
looks good :shipit:

**Pull request merged**